components/MapComponent.tsx

```diff
@@ -35,11 +48,12 @@ interface Props<T> {
 }
 
 // Map view component itself
-export function MapComponent<T extends mapMarker> ({
+export function MapComponent<T extends MapMarker> ({
   initialRegion,
   markers,
+  customizers,
```

Comment:
Not a bad idea to allow each marker to have its own custom style, but passing a customizers array shouldn't be necessary here; the color, icon, etc will be determined from the map marker data.